RELEASE_NOTES.md

```diff
@@ -10,6 +10,108 @@
 Version vxx.yy.zz -- Release date 20yy-mm-dd
 ============================================
```

Comment:
Take this out?

```suggestion
```

Reply:
That's always been in the file. It's a placeholder for starting the next release.